### ISSUE TITLE
Accessibility testing - Process diagram sales data 

### ISSUE THREAD

**Issue by Maroyafaied** [DOI-ONRR/nrrd#3153]
Preview link: https://nrrd-preview.app.cloud.gov/sites/federal-sales-data/downloads/sales-data

This issue is to test and document any accessibility issues with the process diagram. We will then discuss and prioritize the issues since we are on version 4 of Mui there might be some fixes that can't happen until we update to version 5.

**Comment by eelzi-ONRR** [comment]
Light house found the links in the process diagram fail contrast
Trusted tester found the same failed contrast for links, the equations look weird when the page is magnified, the TC table needs a name, the royalty equation is coded as a data table, and decorative svgs need to be hidden.

Closing this issue and we'll discuss with Jeff how he wants us to make issue for the things that need to be fixed
